client/components/chart/index.jsx

```diff
@@ -12,14 +12,18 @@ import { noop, throttle } from 'lodash';
  */
 import BarContainer from './bar-container';
 import { hasTouch } from 'lib/touch-detect';
+import Tooltip from 'components/tooltip';
 
-class Chart extends React.Component {
+class Chart extends React.PureComponent {
```

Comment:
Does this `PureComponent` really prevent any wasted rerender of the chart?

Reply:
Not sure, I actually copied it from #21273 when experimenting with this but then accidentally committed it.

Reply:
It would be nice to check, but not a priority at all.

Reply:
Unfortunately, doesn't help. `render` is still called the same number of times.